kornia/image/image.py

```diff
@@ -160,6 +164,12 @@ def channels_order(self) -> ChannelsOrder:
         """Return the channels order."""
         return self.layout.channels_order
 
+    def get_channel(self, idx: int):
```

Comment:
maybe add a note here to make it work with str -- like:

```python
r = image.get_channel('R')
# maybe in the future
r, b, g = image.get_channel('RGB')
```

Reply:
i think we are gonna drop for now the extension of image api at least for the first iteration